keyboards/plexus75/rules.mk

```diff
@@ -20,3 +20,4 @@ RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
 UNICODE_ENABLE = yes        # Unicode
+VIA_ENABLE = yes	        # Via
```

Comment:
This needs to go at keymap level, not keyboard level (ie. `keymaps/via/rules.mk`).